Jenkinsfile

```
@@ -0,0 +1,88 @@
+pipeline {
+  agent any
+  stages {
+    stage('Verify git commit conventions') {
+      steps {
+        sh 'sh ./gitlint.sh'
+      }
+    }
+    stage('Verify .sh files') {
+      steps {
+        sh 'docker build --target verify-sh .'
+      }
+    }
+    stage('Restore') {
+      steps {
+        sh 'docker build --target restore .'
+      }
+    }
+    stage('Verify Format') {
+      steps {
+        sh 'docker build --target verify-format .'
+      }
+    }
+    stage('Build') {
+      steps {
+        sh 'docker build --target build .'
+      }
+    }
+    stage('Test') {
+      steps {
+        sh 'docker build --target test .'
+      }
+    }
```

Creo que en esta app tendría más sentido correr antes el test que el build

```suggestion
    stage('Test') {
      steps {
        sh 'docker build --target test .'
      }
    }
    stage('Build') {
      steps {
        sh 'docker build --target build .'
      }
    }
```